jira/resources.py

```diff
@@ -953,16 +970,26 @@ def dict2resource(raw, top=None, options=None, session=None):
     r'securitylevel/[^/]+$': SecurityLevel,
     r'status/[^/]+$': Status,
     r'user\?username.+$': User,
+    r'group\?groupname.+$': UserGroup,
     r'version/[^/]+$': Version,
     # GreenHopper specific resources
     r'sprints/[^/]+$': Sprint,
     r'views/[^/]+$': Board}
 
 
+class UnknownResource(Resource):
+    """A JIRA project."""
```

Comment:
This does not seem to be an accurate description of the unknown resource maybe a little explanation why this is here.

Reply:
OK